scheduler/wmap/sample/2.json

```diff
@@ -0,0 +1,35 @@
+{
+    "collect": {
+        "metrics": {
+            "㊽foo㊽bar": {
+                "version": 1
+            },
+            "大foo大baz": {},
+            "/a0/b0": {},
+            "-a1-b1": {},
+            "_a2_b2": {},
+            "㊽a3㊽b3": {},
+            "Äa4Äb4": {},
+            "大a5大b5": {},
+            "小a6小b6": {},
+            "ᵹa7ᵹb7": {},
+            "☍a8☍b8": {},
+            "ヒa9ヒb9": {}
```

Comment:
that's support already. Here is to make sure all chars in the list won't cause issue.